### PR TITLE
Update User Guide with Amos' parts

### PR DESCRIPTION
**Pull request by amosting**
Part of #50 .

This PR updates Amos' parts of the user guide.

### CODE REVIEW

**Comment by LHeng1** [comment]
Seems like the conflict is caused by using the original AB3 User Guide as the skeleton. I have made a skeletal version for our own project recently, maybe you can follow the [forking workflow](https://nus-cs2103-ay2324s1.github.io/website/se-book-adapted/chapters/gitAndGithub.html#forking-workflow) to update your local repo and re-do the task with the latest UG version, or can just solve it on GitHub itself even though it may be a bit confusing, at least for me.

**TLDR for the forking workflow: pull upstream repo to local repo -> do changes -> push to fork -> pull request**

Feel free to close this PR and create a new PR if using the forking workflow method instead!

**Review by LHeng1 — CHANGES_REQUESTED** [review]
Seems like the conflict is caused by using the original AB3 User Guide as the skeleton. I have made a skeletal version for our own project recently, maybe you can follow the [forking workflow](https://nus-cs2103-ay2324s1.github.io/website/se-book-adapted/chapters/gitAndGithub.html#forking-workflow) to update your local repo and re-do the task with the latest UG version, or can just solve it on GitHub itself even though it may be a bit confusing, at least for me.

TLDR for the forking workflow: pull upstream repo to local repo -> do changes -> push to fork -> pull request

Feel free to close this PR and create a new PR if using the forking workflow method instead!

**Comment by ravern** [comment]
Once you have made the necessary changes, you can re-request a review from the reviewer

image

File: docs/UserGuide.md [hunk 1]
```diff
@@ -72,7 +72,52 @@ title: User Guide
 <aside>
 ⭐  Shows a list of all applicants.
 
+<<<<<<< HEAD
 </aside>
+=======
+### d. Adding an applicant: `add`
```
Comment (ravern) [inline docs/UserGuide.md:78]: I don't think this should be included, these are artefacts of a merge conflict

File: docs/UserGuide.md [hunk 2]
```diff
@@ -72,7 +72,52 @@ title: User Guide
...
+**Expected Outputs:**
+
+- The applicant and applicant’s information has been successfully added
+- Confirmation message: **`"Applicant added successfully with the following details: Name: <name>, Phone Number: <phone number>, Email: <email>, GPA: <gpa>, tags: <tags>"`**
+
+**Errors:**
+
+- Missing fields: **`"Error: Missing fields. Please follow the format: add [n/NAME] [p/PHONE] [e/EMAIL] [g/GPA] [t/TAG]…"`**
+- Repeated applicant: **`"Error: Repeated applicant. Applicant has already been added to the list`**
+>>>>>>> user-guide
```
Comment (ravern) [inline docs/UserGuide.md:120]: This too

File: docs/UserGuide.md [hunk 3]
```diff
@@ -92,6 +137,7 @@ title: User Guide
 <aside>
 ⭐ Edits an existing applicant in the list of applicants.
 
+<<<<<<< HEAD
```
Comment (ravern) [inline docs/UserGuide.md:140]: This too

File: docs/UserGuide.md [hunk 4]
```diff
@@ -111,6 +157,49 @@ title: User Guide
     - Edits the phone number and email address of the 1st person to be **`91234567`** and **`johndoe@example.com`**, respectively.
 - **`edit 2 n/Betsy Crower t/`**
     - Edits the name of the 2nd person to be **`Betsy Crower`** and clears all existing tags.
+=======
```
Comment (ravern) [inline docs/UserGuide.md:160]: This too

**Review by ravern — CHANGES_REQUESTED** [review]
Merge conflict artefacts still present

**Review by ylyma — APPROVED** [review]

**Review by ravern — APPROVED** [review]
LGTM

**Review by LHeng1 — APPROVED** [review]
LGTM!
